custom/icds_reports/ucr/data_sources/child_health_cases_monthly_tableau2.json

```diff
@@ -2432,6 +2434,24 @@
           }
         }
       },
+      "child_age_in_days": {

```

Comment:
Is there a benefit to moving this here instead of keeping it in the custom python expressions file?